docs/man/events.1

```diff
@@ -49,8 +49,8 @@ Event return value: Filter events based on their return value using
 The event return value expression follows the syntax of a numerical
 expression.
 .IP \[bu] 2
-Event context fields: Filter events based on the non-argument fields
-defined in the trace.Event struct using `event-name.context.field'.
+Event scope fields: Filter events based on the non-argument fields
+defined in the trace.Event struct using `event-name.scope.field'.
```

Comment:
I think that this assertion can be a bit misleading, I mean many of the former context fields are part of the trace.Event. WDYT?

Reply:
We are renaming context filters to scope filters, and I didn't change anything in this comment.
Isn't it still true that scope filters are to filter non-argument fields in the trace.Event? Why is it misleading?

Reply:
Yep, it's not your changes, I just glanced at that sentence and, thinking as a new user, didn't quickly understand the meaning. We can see how to improve it in the future. It's definitely not related to this PR.